### ISSUE TITLE
Let users cancel a ticket

### ISSUE THREAD

**Issue by Narigo** [compose-us-research/noqueue#22]
**Description**
Change the "Ticket bearbeiten / stornieren" to "Ticket stornieren". Make it work!

**Links/References**
(Add any designs, screenshots or links that helpful to implement the feature)

**Tasks**

- [x] Rephrase "Ticket bearbeiten / stornieren" to "Ticket stornieren"
- [x] Send DELETE request for the ticket
- [x] Make the DELETE request work by flagging the database entry "deleted_at"

**Additional Context**
If there is an infection, we can't completely delete a ticket after using it. Therefore we need to flag the entries instead of fully deleting them.

**Comment by Narigo** [comment]
Blocked by #48 - we need a `deleted_at` column or similar on all ticket tables.

**Comment by Narigo** [comment]
Created new issue #81 for that.
